iocore/net/quic/QUICAltConnectionManager.cc

```diff
@@ -292,10 +293,60 @@ QUICAltConnectionManager::generate_frame(QUICEncryptionLevel level, uint64_t con
   if (!this->_retired_seq_nums.empty()) {
     if (auto s = this->_retired_seq_nums.front()) {
       frame = QUICFrameFactory::create_retire_connection_id_frame(s);
+      this->_records_retire_connection_id_frame(level, *static_cast<QUICRetireConnectionIdFrame *>(frame.get()));
```

Comment:
You can cast to a reference, which might be clearer given the parameter type. E.g.
```
static_cast<QUICRetireConnectionIdFrame&>(*frame)
```

Reply:
updated/